### ISSUE TITLE
Possible Bug based On Code Comments

### ISSUE THREAD

**Issue by amast09** [jaredLunde/react-hook#36]
**Describe the bug**
This isn't necessarily a bug, however I thought I would point it out in case it is unintentional.

The `useTimeoutCallback` function [here]( https://github.com/jaredLunde/react-hook/blob/84466d904a3de2cadf57681fd39598d8854e4a49/packages/timeout/src/index.tsx#L47), only has 2 parameters (`callback` and `ms`) however the code comment [here](https://github.com/jaredLunde/react-hook/blob/84466d904a3de2cadf57681fd39598d8854e4a49/packages/timeout/src/index.tsx#L35) references an additional `dependencies` parameter.

Not sure if this is a code comment that is stale or additional code that is missing.

I hope this was helpful, thanks!

**Comment by jaredLunde** [comment]
You're right, `dependencies` is a stale comment. Will fix it soon. Thanks for the report!
